gbasis/evals/density.py

```diff
@@ -437,6 +483,23 @@ def evaluate_density_hessian(
 ):
     r"""Return the Hessian of the density evaluated at the given points.
 
+    .. math::
+        \begin{equation}
+        H[\rho(\mathbf{r}_n)]
+        =
+        \begin{bmatrix}
+            \frac{\partial^2}{\partial x^2} \rho(\mathbf{r}_n) &
+            \frac{\partial^2}{\partial x \partial y} \rho(\mathbf{r}_n) &
+            \frac{\partial^2}{\partial x \partial z} \rho(\mathbf{r}_n)\\\\
+            \frac{\partial^2}{\partial x \partial y} \rho(\mathbf{r}_n) &
+            \frac{\partial^2}{\partial y^2} \rho(\mathbf{r}_n)&
+            \frac{\partial^2}{\partial y \partial z} \rho(\mathbf{r}_n)\\\\
+            \frac{\partial^2}{\partial x \partial z} \rho(\mathbf{r}_n) &
+            \frac{\partial^2}{\partial z^2} \rho(\mathbf{r}_n)&
+            \frac{\partial^2}{\partial x \partial z} \rho(\mathbf{r}_n)\\
```

Comment:
The variables in the derivatives are not correct. Please double check.

Reply:
Yup, sorry for missing this.